app/assets/stylesheets/components/pill.scss

```diff
@@ -0,0 +1,31 @@
+.component-pill{
+  display: inline-block;
+  background: $primary-color;
+  color: white;
+  padding: 2px 5px 2px 5px;
+  margin-bottom: 5px;
+  border-radius: 10px;
+  i{
+    margin-left: 5px;
+  }  
+}
+.uber_eats{
```

Comment:
Hey @aSadhankar just to make sure we don't accidentally apply these in a different way, I would nest these under the `.component-pill`, so it would look something like this:

```
.component-pill {
  ...
  &.uber_eats {
    background: $uber-green;
  }
}
```